Exiled.Events/Patches/Events/Player/SpawningRagdoll.cs

```diff
@@ -43,6 +43,11 @@ private static bool Prefix(RagdollManager __instance, ref Vector3 pos, ref Quate
                 ownerNick = ev.PlayerNickname;
                 playerId = ev.PlayerId;
 
+                Role role = __instance.hub.characterClassManager.Classes.SafeGet(classId);
+                GameObject gameObject = UnityEngine.Object.Instantiate(role.model_ragdoll, pos + role.ragdoll_offset.position, Quaternion.Euler(rot.eulerAngles + role.ragdoll_offset.rotation));
```

Comment:
Doing this will Firstly, cause the server to instantiate a new ragdoll gameobject even if the event's ev.IsAllowed is set to false.

Secondly, it will cause this event to always instantiate 2 ragdoll game objects, however only one will be spawned into the map. Additionally, the one that is properly spawned in, won't be on your Map.Ragdolls list.

This event needs to be reworked significantly if you are going to accomplish what you think it is you need to be doing here.

Reply:
Alright. I'm gonna do that.